data/colors/themes/dark_high_contrast.ts

```diff
@@ -227,6 +227,9 @@ const exceptions = {
   treeViewItem: {
     chevron: {
       hoverBg: get('scale.gray.6'),
+    },
+    folder: {
+      fill: get('accent.muted')
```

Comment:
I've been calling it `directory` instead of `folder` to align with the octicon naming: https://primer.style/octicons/file-directory-fill-16

```suggestion
    directory: {
      fill: get('accent.muted')
```